packages/astro/src/content/vite-plugin-content-virtual-mod.ts

```diff
@@ -209,5 +210,6 @@ const UnexpectedLookupMapError = new AstroError({
 
 function globWithUnderscoresIgnored(relContentDir: string, exts: string[]): string[] {
 	const extGlob = getExtGlob(exts);
-	return [`${relContentDir}/**/*${extGlob}`, `!**/_*/**${extGlob}`, `!**/_*${extGlob}`];
+	const contentDir = appendForwardSlash(relContentDir.trim());
```

Comment:
Do we need the `trim()` here?

Reply:
I don't know :) Just added as a late sanitization, in case `relContentDir` will ever contain leading or trailing slashes.

Should I remove it?

Reply:
Yeah I don't think it's needed here, we can trust the path that's coming in.

Reply:
Removed it